packages/iris-grid/src/IrisGridMetricCalculator.ts

```diff
@@ -1,5 +1,18 @@
 import { GridMetricCalculator } from '@deephaven/grid';
+import type { VisibleIndex, GridMetricState, GridTheme } from '@deephaven/grid';
 import TableUtils from './TableUtils';
+import type IrisGridModel from './IrisGridModel';
+import type IrisGridTheme from './IrisGridTheme';
+
+export interface IrisGridMetricState extends GridMetricState {
+  model: IrisGridModel;
+  theme: typeof IrisGridTheme & typeof GridTheme;
```

Comment:
Can't. `getGridY` uses `filterBarHeight`, `filterBarCollapsedHeight`, and `reverseHeaderBarHeight` which aren't on the base `GridTheme`.

The reason I intersected the types is to require the types from `GridTheme` since `IrisGridTheme` is a partial w/ extended fields.

Should maybe have the exported theme spread the `GridTheme` object and then provide overrides. Would make the types cleaner since `IrisGridTheme` could extend the `GridTheme` type properly then

Reply:
I meant `typeof IrisGridTheme & GridThemeType`, that should work no?